runtime/v2/runc/container.go

```diff
@@ -506,3 +477,25 @@ func (c *Container) HasPid(pid int) bool {
 	}
 	return false
 }
+
+func loadProcessCgroup(ctx context.Context, pid int) (cg interface{}, err error) {
+	if cgroups.Mode() == cgroups.Unified {
+		g, err := cgroupsv2.PidGroupPath(pid)
+		if err != nil {
+			logrus.WithError(err).WithError(err).Errorf("loading cgroup2 for %d", pid)
+			return nil, err
+		}
+		cg, err = cgroupsv2.LoadManager("/sys/fs/cgroup", g)
```

Comment:
keep using `cgroupsv2.LoadManager("/sys/fs/cgroup"` instead of `cgroupsv2.Load(g)` since 1.6 is still using `containerd/cgroups v1`